### PR TITLE
Add draft get request functions

### PR DESCRIPTION
**Pull request by mrbrendanw**
Does not yet handle bad requests; will need to implement error handling

### CODE REVIEW

**Comment by thethomasz** [comment]
uhh, sure? guess it doesn't really matter?

**Comment by mrbrendanwong** [comment]
No point in using `let` right now since we don't care about scope. If there are concerns, feel free to change

**Comment by BoredX** [comment]
I only used var when I had to at DDS :^ )

**Comment by mrbrendanwong** [comment]
Maybe you just suck tho

**Comment by mrbrendanwong** [comment]
Please review and approve. Code I wrote so far seems like a decent place to start off for y'all.

File: src/data-processor.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    console.log(fileData);
+}
+
+function processBranches(raw) {
+    /*
+    Currently unsure of purpose
+    */
+}
+
+function processCommits(raw) {
```
Comment (mrbrendanwong) [inline src/data-processor.js]: If doids spawn based on file creation in commits, then getContent and processContent may be unnecessary

**Pull request merged**